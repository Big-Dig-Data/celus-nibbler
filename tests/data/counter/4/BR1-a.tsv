Book Report 1 (R4)	Number of Successful Title Requests by Month and Title
Celus University
000000
Period covered by Report:
2018-01-01 to 2018-12-31
Date run:
2020-01-01
	Publisher	Platform	Book DOI	Proprietary Identifier	ISBN	ISSN	Reporting Period Total	Jan-2018	Feb-2018	Mar-2018	Apr-2018	May-2018	Jun-2018	Jul-2018	Aug-2018	Sep-2018	Oct-2018	Nov-2018	Dec-2018
																			
TOTAL FOR ALL TITLES							38	3	2	3	1	10	0	6	0	0	0	0	13
Title1	Publisher1	Ovid			111-1-1111-1111-1		32	3	2	3	1	10	0	6	0	0	0	0	7
Title2	Publisher2	Ovid			222-2-2222-2222-2		6	0	0	0	0	0	0	0	0	0	0	0	6
