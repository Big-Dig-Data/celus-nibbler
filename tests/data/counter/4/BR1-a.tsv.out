"2018-01-01","2018-01-31","","Title1","Book Title Requests","Platform:Ovid|Publisher:Publisher1","DOI:|ISBN:111-1-1111-1111-1|Print_ISSN:|Proprietary:","3"
"2018-02-01","2018-02-28","","Title1","Book Title Requests","Platform:Ovid|Publisher:Publisher1","DOI:|ISBN:111-1-1111-1111-1|Print_ISSN:|Proprietary:","2"
"2018-03-01","2018-03-31","","Title1","Book Title Requests","Platform:Ovid|Publisher:Publisher1","DOI:|ISBN:111-1-1111-1111-1|Print_ISSN:|Proprietary:","3"
"2018-04-01","2018-04-30","","Title1","Book Title Requests","Platform:Ovid|Publisher:Publisher1","DOI:|ISBN:111-1-1111-1111-1|Print_ISSN:|Proprietary:","1"
"2018-05-01","2018-05-31","","Title1","Book Title Requests","Platform:Ovid|Publisher:Publisher1","DOI:|ISBN:111-1-1111-1111-1|Print_ISSN:|Proprietary:","10"
"2018-06-01","2018-06-30","","Title1","Book Title Requests","Platform:Ovid|Publisher:Publisher1","DOI:|ISBN:111-1-1111-1111-1|Print_ISSN:|Proprietary:","0"
"2018-07-01","2018-07-31","","Title1","Book Title Requests","Platform:Ovid|Publisher:Publisher1","DOI:|ISBN:111-1-1111-1111-1|Print_ISSN:|Proprietary:","6"
"2018-08-01","2018-08-31","","Title1","Book Title Requests","Platform:Ovid|Publisher:Publisher1","DOI:|ISBN:111-1-1111-1111-1|Print_ISSN:|Proprietary:","0"
"2018-09-01","2018-09-30","","Title1","Book Title Requests","Platform:Ovid|Publisher:Publisher1","DOI:|ISBN:111-1-1111-1111-1|Print_ISSN:|Proprietary:","0"
"2018-10-01","2018-10-31","","Title1","Book Title Requests","Platform:Ovid|Publisher:Publisher1","DOI:|ISBN:111-1-1111-1111-1|Print_ISSN:|Proprietary:","0"
"2018-11-01","2018-11-30","","Title1","Book Title Requests","Platform:Ovid|Publisher:Publisher1","DOI:|ISBN:111-1-1111-1111-1|Print_ISSN:|Proprietary:","0"
"2018-12-01","2018-12-31","","Title1","Book Title Requests","Platform:Ovid|Publisher:Publisher1","DOI:|ISBN:111-1-1111-1111-1|Print_ISSN:|Proprietary:","7"
"2018-01-01","2018-01-31","","Title2","Book Title Requests","Platform:Ovid|Publisher:Publisher2","DOI:|ISBN:222-2-2222-2222-2|Print_ISSN:|Proprietary:","0"
"2018-02-01","2018-02-28","","Title2","Book Title Requests","Platform:Ovid|Publisher:Publisher2","DOI:|ISBN:222-2-2222-2222-2|Print_ISSN:|Proprietary:","0"
"2018-03-01","2018-03-31","","Title2","Book Title Requests","Platform:Ovid|Publisher:Publisher2","DOI:|ISBN:222-2-2222-2222-2|Print_ISSN:|Proprietary:","0"
"2018-04-01","2018-04-30","","Title2","Book Title Requests","Platform:Ovid|Publisher:Publisher2","DOI:|ISBN:222-2-2222-2222-2|Print_ISSN:|Proprietary:","0"
"2018-05-01","2018-05-31","","Title2","Book Title Requests","Platform:Ovid|Publisher:Publisher2","DOI:|ISBN:222-2-2222-2222-2|Print_ISSN:|Proprietary:","0"
"2018-06-01","2018-06-30","","Title2","Book Title Requests","Platform:Ovid|Publisher:Publisher2","DOI:|ISBN:222-2-2222-2222-2|Print_ISSN:|Proprietary:","0"
"2018-07-01","2018-07-31","","Title2","Book Title Requests","Platform:Ovid|Publisher:Publisher2","DOI:|ISBN:222-2-2222-2222-2|Print_ISSN:|Proprietary:","0"
"2018-08-01","2018-08-31","","Title2","Book Title Requests","Platform:Ovid|Publisher:Publisher2","DOI:|ISBN:222-2-2222-2222-2|Print_ISSN:|Proprietary:","0"
"2018-09-01","2018-09-30","","Title2","Book Title Requests","Platform:Ovid|Publisher:Publisher2","DOI:|ISBN:222-2-2222-2222-2|Print_ISSN:|Proprietary:","0"
"2018-10-01","2018-10-31","","Title2","Book Title Requests","Platform:Ovid|Publisher:Publisher2","DOI:|ISBN:222-2-2222-2222-2|Print_ISSN:|Proprietary:","0"
"2018-11-01","2018-11-30","","Title2","Book Title Requests","Platform:Ovid|Publisher:Publisher2","DOI:|ISBN:222-2-2222-2222-2|Print_ISSN:|Proprietary:","0"
"2018-12-01","2018-12-31","","Title2","Book Title Requests","Platform:Ovid|Publisher:Publisher2","DOI:|ISBN:222-2-2222-2222-2|Print_ISSN:|Proprietary:","6"
